xdsl/parser.py

```diff
@@ -609,7 +608,7 @@ def __init__(self,
                  ctx: MLContext,
                  input: str,
                  name: str = '<unknown>',
-                 allow_unregistered_ops=False):
+                 allow_unregistered_ops: bool = False):
```

Comment:
Do you really have to annotated keyword args? This looks extremely weird. Wait, could you use the variable `allow_unregistered_ops` afterward? Please say no.

Reply:
yeah the type annotations are required for all arguments